test/src/androidTest/kotlin/io/realm/NotificationTests.kt

```diff
@@ -56,6 +56,7 @@ class NotificationTests {
         realm.objects(Sample::class).delete()
         realm.commitTransaction()
         assertEquals(0, realm.objects(Sample::class).size, "Realm is not empty")
+        realm.close()
```

Comment:
Like the tests reflects we have two ways of. closing the Realm
1-  Calling `close` will force close the Realm instance in the current Thread even though references are still being used,  using references (RealmObject passed to ViewModel for instance) after closing the Realm will cause a Segmentation fault
2- The Realm pointer is reference-counted which means that when all the RealmObjects goes out of scope and the Realm instance itself goes out of scope it will be closed